analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ControlCharacterInString.CSharp11.cs

```diff
@@ -4,7 +4,18 @@ namespace Tests.Diagnostics
 {
     class Program
     {
-        public const string Part1 = """test"""; // Noncompliant
-        public const string Part2 = """test""";
+        public const string RawStringLiteralsNonCompliant = """test"""; // Noncompliant
+        public const string RawStringLiteralsCompliant = """test""";
+
+        public const string InterpolatedStringNonCompliant = $"""test{RawStringLiteralsNonCompliant}"""; // FN
+        public const string InterpolatedStringCompliant = $"""test{RawStringLiteralsCompliant}""";
```

Comment:
Besides this case, there should be also one where the control character is in the text part of the interpolated string and not the interpolation.